cmake/extensions.cmake

```diff
@@ -854,7 +854,7 @@ function(zephyr_check_compiler_flag_hardcoded lang option check exists)
   # -Werror=implicit-int is not supported for CXX and we are not able
   # to automatically test for it because it produces a warning
   # instead of an error during the test.
-  if((${lang} STREQUAL CXX) AND ("${option}" STREQUAL -Werror=implicit-int))
+  if((${lang} STREQUAL CXX) AND ("${option}" IN_LIST CXX_EXCLUDED_OPTIONS))
```

Comment:
`# -Werror=implicit-int and other flags in CXX_EXCLUDED_OPTIONS are not supported for CXX and we are not able`

Sorry for the stepwise feedback.

Reply:
Is the new wording OK?  I didn't want to mention specific flags because they might not apply to certain toolchains.

Reply:
Looks great